cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractApiTest.java

```diff
@@ -181,10 +181,11 @@ public void test03ResourcesCursor() throws Exception {
     @Test
     public void test04ResourcesByType() throws Exception {
         // should allow listing resources by type
-        Map resource = preloadResource(ObjectUtils.asMap("tags", UPLOAD_TAGS));
-        Map result = api.resources(ObjectUtils.asMap("type", "upload", "max_results", 500));
+        Map result = api.resources(ObjectUtils.asMap("type", "upload", "max_results", 10));
```

Comment:
@nitzanj , is that related to the current change?

Reply:
Not really, but through this PR I found out there was a reliability issue with timing and resource-flooded test account. 
However the test fixes are on their own commit so the history will remain clean, on master too.

Reply:
Why not having 2 separate PRs?

Reply:
Quicker and no merges, and it ends up identical on master's history when merged.

Reply:
Approving, but let's keep PRs as focused as possible in the future.